3741/1/lib/check_html.rb

```diff
@@ -0,0 +1,11 @@
+require 'json'
+# check url validate and return base html or nil object
+class CheckHtml
```

Comment:
Class should be noun, method should be verbs.
HtmlChecker

Reply:
UrlFetcher

Reply:
UriFetcher

Reply:
refactor this on